crt_portal/cts_forms/templates/forms/complaint_view/show/activity_stream.html

```diff
@@ -0,0 +1,24 @@
+<div class="complaint-card complaint-card--rounded">
+  <div class="activity-stream">
+    <div class="heading">
+      Activity
+    </div>
+    <div class="complaint-card-content">
+      <ul class="activity-stream-list usa-list usa-list--unstyled">
+        {% for activity in activity_stream %}
+          <li class="activity-stream-item">
+            <div class="display-flex flex-justify">
+              <b  >{{activity.actor.email}}</b>
```

Comment:
Because of how SSO works, we will have username auto-populated but not email. This will ensure we hae a name for everyone. 
```suggestion
              <b  >{{activity.actor.username}}</b>
```

Reply:
Good to know!